_posts/2022-01-14-quorn-a-story-about-single-cell-protein.md

```diff
@@ -0,0 +1,197 @@
+
+Quorn doesn't really have a protein conversion efficiency because it directly utilizes ammonia, not protein, as a nitrogen source. 
+
+##### Quorn as a Pathway
+
+Having a direct pathway from **carbohydrates + ammonia** to **protein** has a lot of neat implications. If plant-based meat substitutes continue to gain traction (and I think they will), the production of plant protein isolates and concentrates is going to yield a lot of waste carbohydrates and starches, which will become dirt cheap. This would make the Quorn process a lucrative pathway to upgrade cheap carbohydrates into protein.
+
+Alternatively, if corn ethanol gets banned due to policy changes, we're going to have a lot of un-used corn milling and hydrolysing infrastructure. These could easily be adapted into Quorn-type protein production plants. 
+
+**Taste and Texture Matter**
+
+A food products' **organoleptic properties** matter a lot in the meat substitute space. Quorn's success comes in part due to the fact the the hyphal product, after heat treatment and freezing, resembles the fibrous texture of meat. 
+
+![](/assets/images/quorn/quorn-meatless-pieces-fried-texture.png){:height="350" .center}
+*Fried Quorn nuggets - Source: veggiepattytastetest.com*
+
+
+##### Quorn as a Miso
+
+I'd love to get my hands on some *F. venenatum* biomass, centrifuged, **without heat-treating to remove RNA**. I would take this 20-30% solids biomass and heat treat it myself, keeping the 8-9% nucleic acids, and maybe make a miso or garum out of it, since it would contain plenty of natural 5'-ribonucleotides and glutamates. In theory it would make an umami bomb that doesn't have the flavour of yeast extracts. At the small amounts I would be consuming in either form, uric acid isn't a problem: so long as daily consumption is under the WHO recommendation of <2g RNA per day (a tablespoon of this stuff should contain around 0.3g)
+
+Maybe I'll ask Quorn for some, or make my own. The original patents for this bioprocess have already expired - though it would be difficult to replace Marlow Foods' 35 years of operating experience. 
+
+##### The Protein Crisis 
+
+The original goal at inception, to prevent protein malnutrition, was never really met - because thankfully, and generally speaking - the 'Protein Crisis' neve happened agricultural productivity kept up with population growth through the 1960s, 70s, 80s, and beyond. This was mostly due to the *Green Revolution* consisting of better crop strains, fertilizer use, and the like. SCP could never compete economically with plant proteins. 
+
+But I can't fault the authors of the original UN paper: in 1967 we hadn't even discovered RuBisCO, a key enzyme behind photosynthesis. This discovery wouldn't be made until well into the 1980s [12]. Before that, it might have been safe to assume that plants simply assimilate nitrogen to create protein, so we could bypass the process altogether by adding sugar and nitrogen into a bioreactor.
```

Comment:
it took me a few tries to understand the last sentence here. The next paragraph slightly clears it up, but I'm still a bit muddled about the point being made. If I understand correctly, maybe something like:

> But I can't fault the authors of the original UN paper: in 1967 we hadn't even discovered RuBisCO, a key enzyme behind carbon-fixation in plants (this discovery wouldn't be made until well into the 1980s [12]). Before that, it might have been safe to assume that plants simply assimilate nitrogen to create protein. Perhaps it was imagined that we could bypass the costly "work" of creating plant fibers, and focus solely on protein creation by adding sugar and nitrogen into a bioreactor. As it turned out, even with the inefficiencies of producing plant fiber (of which little can be upcycled), it was still more efficient to use plants.